cibyl/sources/source.py

```diff
@@ -80,6 +80,11 @@ def setup(self):
         """Setup everything required for the source to become operational."""
         pass
 
+    @abstractmethod
+    def _validate(self):
+        """Validate things which we have setup for source."""
+        pass
```

Comment:
A private abstract method is meant only for the base class to call, so that each child can specialize a certain functionality from the parent. This is not the case, so I think we could remove this one.